client/coral-admin/src/graphql/utils.js

```diff
@@ -66,21 +66,27 @@ function addCommentToQueue(root, queue, comment, sort) {
 
 function getCommentQueues(comment) {
   const queues = ['all'];
+
   if (comment.status === 'ACCEPTED') {
     queues.push('accepted');
   }
   else if (comment.status === 'REJECTED') {
     queues.push('rejected');
   }
-  else if (comment.status === 'PREMOD') {
-    queues.push('premod');
-  }
-  if (
+  else if (
     ['NONE', 'PREMOD'].indexOf(comment.status) >= 0
     && comment.actions && comment.actions.some((a) => a.__typename === 'FlagAction')
   ) {
     queues.push('flagged');
   }
+  else if (comment.status === 'PREMOD') {
+    queues.push('premod');
+    queues.push('new');
+  } 
+  if (comment.status === 'NONE') {
+    queues.push('new');
+  }
```

Comment:
Are those changes above intentional? It means that if a comment is `flagged` it will not appear in the `premod` nor in the `new` queue. It essentially means if a comment gets flagged, it will be removed from the `premod` and from the `new` queue, which I don't think is good, because flags are made by users and I don't want users to move stuff around in `premod` and in the `new` queue.